capture/config.default.json

```diff
@@ -2,20 +2,23 @@
 	"viewports" : [
 		{
 		 "name": "phone",
-		 "viewport": {"width": 320, "height": 480}
+		 "width": 320,
+     "height": 480
```

Comment:
@jehoshua02     Please go ahead and format this file.   I will pull your changes.